src/System.Private.ServiceModel/tests/Scenarios/Binding/Custom/Binding.Custom.Tests.csproj

```diff
@@ -17,27 +17,27 @@
     <ProjectGuid>{EDC88742-E7BF-4B7A-9DE2-55055B81B8F2}</ProjectGuid>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs" />
+    <Compile Include="*.cs" />
```

Comment:
Nit: in general I prefer to use the "**\*.cs pattern so that we can add subfolders easily without touching the csproj.  But not blocking if you go ahead and make this change.